### PR TITLE
fixed validation error on resources with helm-2.14/kubectl-1.14

### PR DESCRIPTION
**Pull request by mteodori**
launch with helm 2.14 which uses the kubectl 1.14 lib:

Error: error validating "": error validating data: [ValidationError(Deployment.spec.template.spec.containers[0].livenessProbe): unknown field "limits" in io.k8s.api.core.v1.Probe, ValidationError(Deployment.spec.template.spec.containers[0].livenessProbe): unknown field "requests" in io.k8s.api.core.v1.Probe]

### CODE REVIEW

**Comment by mteodori** [comment]
fixes also #207 and #208

**Comment by killerboot** [comment]
The issue will be fixed and tracked in https://issues.alfresco.com/jira/browse/REPO-4513

File: helm/alfresco-content-services/templates/deployment-transform-router.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: apps/v1
```
Comment (gavincornwell) [inline helm/alfresco-content-services/templates/deployment-transform-router.yaml:2]: Is this a typo? I presume we need to keep the "kind: Deployment" and change the previous line!?

Comment (mteodori) [reply helm/alfresco-content-services/templates/deployment-transform-router.yaml:2]: yes definitely

**Pull request merged**